packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -613,6 +613,20 @@ export class MergeTree {
 		this.attributionPolicy = options?.attribution?.policyFactory?.();
 	}
 
+	public mintNextLocalOperationStamp(): OperationStamp {
```

Comment:
nit: i think this function could be better on the collab window itself, and then we could remove outside mutability of local seq.